api.py

```diff
@@ -36,6 +37,19 @@ async def validate_schema_request_body(payload=Body(None)):
 async def validate_schema_from_url(url=None):
     if url:
         logger.info("Validating schema from URL", url=url)
+        parsed_url = urlparse(url)
+        if not is_hostname_allowed(parsed_url.hostname):
+            return Response(
+                status_code=400,
+                content=f"URL domain [{parsed_url.hostname}] is not allowed",
+            )
+        if parsed_url.hostname == "github.com" and not parsed_url.path.startswith(
```

Comment:
I don't think it is possible to validate the schema from github without it being the raw json or a gist? So not sure this if ever applies?
```
if parsed_url.hostname == "github.com" and not parsed_url.path.startswith(
    "/ONSdigital/"
):
```

Reply:
I have removed the github.com check since it is not needed anymore as we cannot validate via url with a github.com domain.
I have also changed the domains from a list to a set.
🫡